plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -61,9 +61,6 @@
         id="com.google.cloud.tools.eclipse"
         name="Google Cloud Platform"
         class="com.google.cloud.tools.eclipse.properties.GooglePropertyPage">
-       <enabledWhen>
-         <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
```

Comment:
If we show it on all projects, then we can work around the problem that if a project does not have a GAE facet, then the deployment preferences item won't be shown on the left. If the user opens the preferences and adds either of the GAE facets the deployment preferences will not appear.

(You're correct about the `onlyInGCPProjects`'s incorrect naming.)